e2e-tests/README.md

```diff
@@ -41,3 +41,13 @@ for e2e tests in the pipelines. To create the required files locally run this co
 ````
 ./create-large-test-files.sh
 ````
+
+#### Integration with ACP
```

Comment:
`#### Integration with Appeals Casework Portal (ACP)`

i did not understand what ACP meant

Reply:
OK. I have updated the readme to include the full name and the url link.